source/configuration_manual/config_file/config_variables.rst

```diff
@@ -162,29 +162,74 @@ Authentication variables:
 |          |                     |                                                               |
 |          |                     | .. versionadded:: v2.2.26                                     |
 +----------+---------------------+---------------------------------------------------------------+
-| %l       | lip                 | local IP address                                              |
+| %l       | local_ip            | local IP address                                              |
+|          |                     |                                                               |
+|          |                     | .. versionadded:: v2.3.12 For older versions use %{lip}       |
```

Comment:
do we have a rule for when to use versionadded vs versionchanged?

Reply:
or, should we list %{lip} as versionremoved?

Reply:
I don't think there's "versionremoved"? There is "depracated" which I added. (And it is just deprecated, not removed.)

Anyway .. I'm not sure about versionadded vs versionchanged. I'm thinking maybe it's clearer to have the old and new fields separately listed. With versionchanged I think we'd have to keep only the new field in the table, and use something like ".. versionchanged:: v2.3.12 The long name was changed. The old long name "lip" is deprecated."

Reply:
@cmouse @thuovila Any thoughts?

Reply:
I think ..versionchanged makes more sense. It is is bit annoying there is no ..versionremoved.

Reply:
or, well, https://pypi.org/project/sphinx-removed-in/ we could include this?